### ISSUE TITLE
submit-success with tag template not work in amp-list

### ISSUE THREAD

**Issue by chiqui3d** [ampproject/amphtml#25790]
## What's the issue?

The problem is that I need an initial state for a form, in this case **star rating**, I realized and after following instructions that I had to put the form into the `amp-list` component to get it, I finally got it, the problem comes that I need to show the answer of the form, those answers I have done externally following the documentation, because putting it into the `amp-list template` does not work the nested, doing it externally as in this example does not work either:

## How do we reproduce the issue?

```html
<amp-state id="ratingValues" credentials="include" src="//mydomain.com/get-rating-by-post/8">
</amp-state>
<amp-list
  class="amp-wrapper-current-rating"
  layout="fixed-height"
  height="20"
  credentials="include"
  items="."
  single-item
  noloading
  src="//mydomain.com/get-rating-by-post/8"
  binding="always">
  <template type="amp-mustache">
    <div class="wrapper-current-rating">
          <p>
            La valoración actual: <b><span [text]="ratingValues.rating ? ratingValues.rating : ''">{{rating}}</span></b>
          </p>
    </div>
    <div class="rating-static">
      <p style="margin-bottom: 0;line-height: 1">Valorar articulo:</p>
      <form id="rating"
            method="post"
            action-xhr="//mydomain.com/save-rating-by-post/8"
            target="_blank"
            on="submit-success:AMP.setState({ratingValues: {rating: event.response.mediaRating}})">
        <input name="blog_id" type="hidden" id="blog_id" value="{{ blog.id }}">
        <fieldset class="rating">
          <input name="rating"
                 type="radio"
                 id="rating5"
                 value="5"
                 on="change:rating.submit"
                 [checked]="(ratingValues.rating > 4 && ratingValues.rating <= 5) ? 'checked' : ''" checked>
          <label for="rating5"
                 title="5 stars">☆</label>
          <input name="rating"
                 type="radio"
                 id="rating4"
                 value="4"
                 on="change:rating.submit"
                 [checked]="(ratingValues.rating > 3 && ratingValues.rating <= 4) ? 'checked' : ''" checked>
          <label for="rating4"
                 title="4 stars">☆</label>
          <input name="rating"
                 type="radio"
                 id="rating3"
                 value="3"
                 on="change:rating.submit"
                 [checked]="(ratingValues.rating > 2 && ratingValues.rating <= 3) ? 'checked' : ''" checked>
          <label for="rating3"
                 title="3 stars">☆</label>
          <input name="rating"
                 type="radio"
                 id="rating2"
                 value="2"
                 on="change:rating.submit"
                 [checked]="(ratingValues.rating > 1 && ratingValues.rating <= 2) ? 'checked' : ''" checked>
          <label for="rating2"
                 title="2 stars">☆</label>
          <input name="rating"
                 type="radio"
                 id="rating1"
                 value="1"
                 [checked]="(ratingValues.rating > 0 && ratingValues.rating <= 1) ? 'checked' : ''" checked>
          <label for="rating1"
                 title="1 stars">☆</label>
        </fieldset>
        <div submit-success template="submit_success_rating_template"></div>
        <div submit-error template="submit_error_rating_template"></div>
      </form>
    </div>
  </template>
</amp-list>
<template type="amp-mustache" id="submit_success_rating_template">
   <p style="margin-bottom:0;">Gracias por valorar con <b>{{rating}}</b> estrella(s)!</p>
</template>
<template type="amp-mustache" id="submit_error_rating_template">
    <p>Error: {{message}}</p>
</template>
```
As you can see when I mean that I have to do the answer externally from the form is as follows

```html
.....
<div submit-success template="submit_success_rating_template"></div>
<div submit-error template="submit_error_rating_template"></div>
      </form>
    </div>
  </template>
</amp-list>
<template type="amp-mustache" id="submit_success_rating_template">
   <p style="margin-bottom:0;">Gracias por valorar con <b>{{rating}}</b> estrella(s)!</p>
</template>
<template type="amp-mustache" id="submit_error_rating_template">
    <p>Error: {{message}}</p>
</template>
```
I have realized that if I take the form out of the component `amp-list`, it works. The problem is that I need it to be in the `amp-list` because I need an `initial status`. Any help or recommendation would be welcome.

## What browsers are affected?

Windows 10
Chrome Versión 78.0.3904.108 (Build oficial) (64 bits)

## Which AMP version is affected?

Version 1911121900560

**Comment by mrtnbrjs** [comment]
Did you resolved this issue? I have the same problem

**Comment by caroqliu** [comment]
Thanks for posting this issue. Is it possible with your setup to initialize the form from the URL? You may be able to use the [`data-initialize-from-url`](https://amp.dev/documentation/components/amp-form/#data-initialize-from-url) attribute on `amp-form` instead.

**Comment by chiqui3d** [comment]
Thanks @caroqliu, but having to put the rating of every post in the URL, I find it horrible. The truth of this AMP, is to reinvent the wheel badly made.

**Comment by chiqui3d** [comment]
I'm going to try with amp-script,

**Comment by chiqui3d** [comment]
I haven't done it yet, the client hasn't asked me about it in a while and I haven't done it.

**Comment by caroqliu** [comment]
Is it possible to nest with this structure:
```
<form>
  <amp-list>
    <template>
    ... inputs
    </template>
  </amp-list>
  <div submit-success></div>
  <div submit-error></div>
</form>
```

**Comment by caroqliu** [comment]
I'm going to close the issue for now, please feel free to re-open if you run into additional challenges in your implementation.
